### DISCUSSION TITLE
[BUG] using FilterVariant MultiSelect will always show the filter icon, even if nothing is selected

### DISCUSSION THREAD

**Discussion by BennyAlex** [KevinVandy/material-react-table#539] [General]
@KevinVandy 

![image](https://github.com/KevinVandy/material-react-table/assets/17402724/3d98d1e3-7b63-4346-b4a2-aa5f55763e93)

https://codesandbox.io/s/sad-estrela-k5d355?file=/src/JS.jsx:1061-1094

When opening this page, the filter icon is already visible and showing undefined, even there is nothing selected.

**Comment by vaidasbl** [comment]
Having the same issue
